### DISCUSSION TITLE
Sumatra PDF REader - Textmarker (Yellow)

### DISCUSSION THREAD

**Discussion by novo53** [sumatrapdfreader/sumatrapdf#2604] [Ideas]
The new release 3.4 ist much more perfect now.  Only one function where missing.
A yellow Textmarker funktion wil be appriciatet.

**Comment by GitHubRulesOK** [comment]
Text Marking should be press A on a selection
![image](https://user-images.githubusercontent.com/12754913/170708447-5858254a-7309-44c1-a34c-5af417043fc2.png)

For an area use Ctrl +Left Mouse to select area then A to mark text area, however MuPDF uses an odd way to mark rectangles

![image](https://user-images.githubusercontent.com/12754913/170709256-4acc0aa4-0423-4f9f-ace4-1b47593cbf53.png)


**Best to select multiple lines at the same time.** But they count as one highlight so cannot remove just one.

![image](https://user-images.githubusercontent.com/12754913/170709770-503a4e06-26d9-4248-9b46-33a58d5046ea.png)
